nats.go

```diff
@@ -2457,6 +2475,9 @@ func (nc *Conn) processInfo(info string) error {
 	// did not include themselves in the async INFO protocol.
 	// If empty, do not remove the implicit servers from the pool.
 	if len(ncInfo.ConnectURLs) == 0 {
```

Comment:
The connectURLs is 0 here? Shouldn't it be everyone but that server?

Reply:
This check was there before. You have to think of the case say of single server or where client advertise has been disabled. There is still value/need to react to an LDM notification.

Reply:
Do we do the LDM callback twice then?

Reply:
No, this `if` used to simply `return nil`, now I just check for LDM and handler and invoke, but still `return nil` at line 2481.